gentoo-ci/gentoo-ci.bash

```diff
@@ -37,6 +40,9 @@ if [[ ${PREV_COMMIT} != ${CURRENT_COMMIT} ]]; then
 	"${PKGCHECK_RESULT_PARSER_GIT}"/pkgcheck2borked.py \
 		-x "${PKGCHECK_RESULT_PARSER_GIT}"/excludes.json \
 		-s -w -o warning.list *.xml
+
+	# Sort XML for better Git delta compression
+	xsltproc "${selfdir}/../sort-output.xsl" output.xml | sponge output.xml
```

Comment:
What's `sponge` and which package provides it?

Reply:
From debian's `moreutils`; like `tee`, but only creates the new file when the stdin  closes; otherwise a pipe can clobber the input file. Alternative would be using another file here and `mv`

Reply:
So you're introducing a nonstandard dependency just to avoid using a temporary file?

Reply:
fine, dropped that